posts/views.py

```diff
@@ -71,12 +71,14 @@ def set_post_extra(post, request):
     can_be_edited = post.original_or_self().author.pk == request.user.pk
     is_favorited = post.original_or_self().favorites.filter(pk=request.user.pk).exists()
     can_be_deleted = post.author.pk == request.user.pk
+    number_of_favorites = len(post.favorites.all())
```

Comment:
Use a `COUNT` query instead of getting all data from the DB just to get the length.